### ISSUE TITLE
Scale extension is wrong for scaleByFactor (Req13:Test scale-axes-by-factor result) in test script 

### ISSUE THREAD

**Issue by bangph** [opengeospatial/ets-wcs20#70]
I cannot reopen a closed issue from https://github.com/opengeospatial/ets-wcs20/issues/43. But this problem is existing.

The WCS standard OGC 12-039 states:

> Requirement 5 getCoverage-scale-by-factor-positive:
The Scal::scaleByFactor parameter in a GetCoverage? request, if present, shall have a
positive float value.
Note A value of 1.0 leaves the coverage unscaled, a value between 0 and 1 scales down (reduces target domain), a value greater than 1 scales up (enlarges target domain).

However, **in Req13:Test scale-axes-by-factor with scaleByFactor = 2**  
`http://ENDPOINT?CoverageId=test_mean_summer_airtemp&REQUEST=GetCoverage&SCALEFACTOR=2.0&SERVICE=WCS&VERSION=2.0.1`

**it checks that the result should be divided by 2 (div 2) instead of multiple by 2 (mul 2)**.
from http://cite.opengeospatial.org/teamengine/listings/srv_local_teamengine-prod_TE_BASE_scripts_wcs_2.0.1_ctl_ext_get-scal.html#wcs2:get-scal-req13

> 
	<ctl:test name="wcs2:get-scal-req13">
		<ctl:param name="GetCoverage"/>
		<ctl:param name="DescribeCoverage"/>
		<ctl:assertion>Req13:Test scale-axes-by-factor result.</ctl:assertion>
		<ctl:code>
			<xsl:if test="not(floor(number(tokenize(string($DescribeCoverage//*[local-name()='domainSet']//*[local-name()='GridEnvelope']/*[local-name()='high']),'\s+')[1]) div 2) = number(tokenize(string($GetCoverage//*[local-name()='domainSet']//*[local-name()='GridEnvelope']/*[local-name()='high']),'\s+')[1] ))">
				<ctl:message>[FAILURE] The scal result is not correct.</ctl:message>
				<ctl:fail/>
			</xsl:if>
		</ctl:code>
	</ctl:test>

Endpoint to test: http://conformance.rasdaman.com/ows

This was fixed in here though https://github.com/opengeospatial/ets-wcs20/blob/master/src/main/scripts/ctl/ext_get-scal.xml (search for get-scal-req13)

**Comment by bangph** [comment]
@dstenger @lgoltz it has been not fixed yet https://github.com/opengeospatial/ets-wcs20/issues/43#issuecomment-369281258

**Comment by pebau** [comment]
basically, we have 2 options:
- correct ETS to comply with the standard
- generate a corrigendum to the standard.
If the latter is felt desirable I can initiate that and propose a corrigendum to WCS.SWG at the next TC meeting in October.

**Comment by lgoltz** [comment]
@bangph Can you check this issue again?

As far as I see **multiple by 2 (mul 2)** is used: https://github.com/opengeospatial/ets-wcs20/blob/80031ab5ef7233b58468ba1977b40e62bcc0b30c/src/main/scripts/ctl/ext_get-scal.xml#L2099

**Comment by bangph** [comment]
> @bangph Can you check this issue again?
> 
> As far as I see **multiple by 2 (mul 2)** is used:
> 
> https://github.com/opengeospatial/ets-wcs20/blob/80031ab5ef7233b58468ba1977b40e62bcc0b30c/src/main/scripts/ctl/ext_get-scal.xml#L2099

@lgoltz no, it has not fixed in http://cite.opengeospatial.org/teamengine/listings/srv_local_teamengine-prod_TE_BASE_scripts_wcs_2.0.1_ctl_ext_get-scal.html#wcs2:get-scal-req13 
Can you update it to *mul* instead of *div*

**Comment by lgoltz** [comment]
@bangph #43 was fixed for version 1.13 of the ETS. You tested on July 15th, at this time 1.12 was installed on production. Since July 19th 1.14 is installed. The test should be fixed on production now.

**Comment by bangph** [comment]
@lgoltz Did you see the link I posted and search for `div 2?`
http://cite.opengeospatial.org/teamengine/listings/srv_local_teamengine-prod_TE_BASE_scripts_wcs_2.0.1_ctl_ext_get-scal.html#wcs2:get-scal-req13

**Comment by pebau** [comment]
please hold this for a while. At the last TC meeting an update was approved for WCS Scaling, but there is still some discussion pending. I'd come back as soon as the sky is clear on this.

**Comment by lgoltz** [comment]
@bangph Yes, I did. But it seems that this file was not updated (or removed) at the installation on July 19th. In the ctl script on the server `mul` is used instead of `div` as expected. 
I've tested with http://conformance.rasdaman.com/ows?service=WCS&request=GetCapabilities&acceptversions=2.0.1 and the test is successful. 

@pebau Ok, but I would prefer to close this ticket and open a new one if any changes are required.
